Spec/RestClient.swift

```diff
@@ -1632,14 +1632,10 @@ class RestClient: QuickSpec {
                         waitUntil(timeout: testTimeout) { done in
                             do {
                                 try rest.request("get", path: "feature", params: nil, body: nil, headers: nil) { paginatedResponse, error in
-                                    guard let error = error else {
-                                        fail("Error is empty"); done(); return
-                                    }
+                                    expect(error).to(beNil())
                                     guard let paginatedResponse = paginatedResponse else {
                                         fail("PaginatedResult is empty"); done(); return
                                     }
-                                    expect(error.statusCode) == 404
```

Comment:
Why are there no corresponding assertions for the properties of `paginatedResponse` ?

Reply:
@paddybyers There already exist at the bottom. Did you miss them? https://github.com/ably/ably-ios/blob/63ebbf5a24d5807a19e07ddbfb2de0e9e65def24/Spec/RestClient.swift#L1642-L1647

Reply:
Yes :(

Reply:
@paddybyers 😄 So we're good?